docs/pages/reference/cli.mdx

```diff
@@ -2634,7 +2634,8 @@ tctl version
 
 ## Resource filtering
 
-Both `tsh` and `tctl` allow you to filter `nodes`, `apps`, `db`, and `kube` resources using the `--search` and `--query` flags.
+Both `tsh` and `tctl` allow you to filter `nodes`, `apps`, `db`, and `kube`
```

Comment:
Should we add `windows_desktop` to this list?

Reply:
Added. I've also used regular English instead of code style for these terms since `apps`, `nodes`, etc. aren't actually mentioned in the `--search` flag.